### PR TITLE
add and style hero title & subtitle

### PR DESCRIPTION
**Pull request by jibranabsarulislam**
#117 

https://github.com/stride3d/stride-website/assets/70596906/3906ceff-8dee-4ff6-9b2b-385857c9dea5

### CODE REVIEW

**Comment by jibranabsarulislam** [comment]
@dot-net-policy-service agree

File: css/styles.scss [hunk 1]
```diff
@@ -35,6 +35,70 @@ a {
...
+
+        a {
+            margin-top: 10px;
+        }
+    }
+
+}
+
+.hero-title {
+    font-style: "Roboto", sans-serif;
```
Comment (VaclavElias) [inline css/styles.scss:75]: This should be font-family or not needed to keep the consistency of font?

File: css/styles.scss [hunk 2]
```diff
@@ -35,6 +35,70 @@ a {
...
+    }
+
+    font-weight: 700;
+    color: #fff;
+    text-shadow: 2px 2px 8px #0000001a;
+    white-space: nowrap;
+}
+
+.hero-subtitle {
+    font-style: "Roboto", sans-serif;
```
Comment (VaclavElias) [inline css/styles.scss:89]: This should be font-family or not needed to keep the consistency of font?

File: css/styles.scss [hunk 3]
```diff
@@ -35,6 +35,70 @@ a {
...
+.carousel-caption {
+    position: absolute;
+    bottom: 20px;
+    left: 0;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: end;
+    // margin-bottom: 20px;
```
Comment (VaclavElias) [inline css/styles.scss:47]: Can be removed?

File: css/styles.scss [hunk 4]
```diff
@@ -35,6 +35,70 @@ a {
...
+    z-index: 2;
+    height: 100%;
+    // background-color: red;
```
Comment (VaclavElias) [inline css/styles.scss:50]: Can be removed?

File: css/styles.scss [hunk 5]
```diff
@@ -35,6 +35,70 @@ a {
...
+    font-style: "Roboto", sans-serif;
+    font-size: 3rem;
+
+    @media screen and (max-width: 768px) {
+        font-size: 2rem;
+    }
+
+    font-weight: 700;
+    color: #fff;
+    text-shadow: 2px 2px 8px #0000001a;
```
Comment (VaclavElias) [inline css/styles.scss:84]: I am not css expert, does it do anything in our case?

**Comment by VaclavElias** [comment]
@jibranabsarulislam, I have added some comments. Would you be ok to fix it? If not, don't worry I can merge and update myself later :)

**Comment by jibranabsarulislam** [comment]
> @jibranabsarulislam, I have added some comments. Would you be ok to fix it? If not, don't worry I can merge and update myself later :)

on it, thank you @VaclavElias ^^

**Comment by VaclavElias** [comment]
Thank you @jibranabsarulislam

**Pull request merged**